spec/operators/startWith-spec.ts

```diff
@@ -143,21 +142,6 @@ describe('startWith', () => {
     });
   });
 
-  it('should allow unsubscribing explicitly and early', () => {
-    testScheduler.run(({ hot, expectObservable, expectSubscriptions }) => {
-      const e1 = hot('  ---a--b----c--d--|');
-      const unsub = '   ---------!        ';
-      const e1subs = '  ^--------!        ';
-      const expected = 's--a--b---        ';
-      const values = { s: 's', a: 'a', b: 'b' };
-
-      const result = e1.pipe(startWith('s', testScheduler));
```

Comment:
Just a reinsurance. Are you sure that these tests should be deleted? Isn't it enough just to remove `testScheduler`?

Reply:
Yeah, they're invalid tests now. This would test that it would start with `'s'` and then an instance of `testScheduler`. :)